### ISSUE TITLE
Hammerhead crashes if DOM contains iframes with quotes in name attribute

### ISSUE THREAD

**Issue by martinkutter** [DevExpress/testcafe#7194]
### What is your Scenario?

DOM contains an iframe like 
```
<iframe src="https://example.com" name='my "example" page'/>
```

### What is the Current behavior?

If i run any test, hammerhead crashes with: Failed to execute 'querySelectorAll' on 'Document': '*[target="my "example" page"]' is not a valid selector.

### What is the Expected behavior?

hammerhead handles this special case of name attributes correct

### What is your public website URL? (or attach your complete example)

https://crs7fr.csb.app/

### What is your TestCafe test code?

```
fixture(`Iframe`).page("https://crs7fr.csb.app/");

test("iframe input exists", async (t) => {
    await t.expect(Selector(`[data-test-id="iframe"]`).count).eql(1);
});
```

### Steps to Reproduce

1. run example test provided above

### TestCafe version

1.18.4

### Command-line arguments

-

**Comment by martinkutter** [comment]
Could be fixed by changing https://github.com/DevExpress/testcafe-hammerhead/blob/master/src/client/sandbox/node/element.ts#L960 to

`const elementsWithTarget = nativeMethods.querySelectorAll.call(this.document, `*[target="${iframe.name.replaceAll('"', '\\"')}"]`); `

**Comment by AlexKamaev** [comment]
I was able to reproduce the issue. Moreover, I confirm that your fix solves the described problem. If you want, you can create a PR with the suggested fix. We will review it.

**Comment by martinkutter** [comment]
fixed by #2788
